src/core/js/router.js

```diff
@@ -207,45 +216,43 @@ define([
         navigateToArguments: function(args) {
             args = this.pruneArguments(args);
 
+            var options = { trigger: false, replace: false };
+
             switch (args.length) {
                 case 0:
-                    this.navigate("#/", {trigger:false, replace:false});
+                    this.navigate('#/', options);
                     break;
                 case 1:
                     if (Adapt.findById(args[0])) {
-                        this.navigate("#/id/"+args[0], {trigger:false, replace:false});
+                        this.navigate('#/id/' + args[0], options);
                     } else {
-                        this.navigate("#/"+args[0], {trigger:false, replace:false});
+                        this.navigate('#/' + args[0], options);
                     }
                     break;
                 case 2:
-                    this.navigate("#/"+args[0]+"/"+args[1], {trigger:false, replace:false});
-                    break;
                 case 3:
-                    this.navigate("#/"+args[0]+"/"+args[1]+"/"+args[2], {trigger:false, replace:false});
+                    this.navigate('#/' + args.join('/'), options);
                     break;
+                default:
+                    this.handleRoute.apply(this, args);
```

Comment:
Are you using this function? @brian-learningpool cos it really needs to be removed.
I would prefer a deprecated warning at the top of this one.

Reply:
No, I'm not using it for anything, @oliverfoster.  I'll add a `warn`.

Reply:
Cool. just something like 'use Backbone.history.navigate or window.location.href instead of Adapt.trigger("router:navigateTo");'